src/components/LegacyTooltip/LegacyTooltip.tsx

```diff
@@ -315,6 +317,7 @@ export const LegacyTooltip = ({
                     className={merge([
                         'tw-popper-container tw-inline-block tw-max-w-[200px] tw-dark tw-bg-base tw-rounded-md tw-shadow-mid tw-text-text tw-z-[120000]',
                         !isOpen && 'tw-opacity-0 tw-h-0 tw-w-0 tw-overflow-hidden',
+                        withStrongBorder && 'tw-border tw-border-line-strong',
```

Comment:
Since there's no other border option we could just use 'border' or 'withBorder'

Reply:
As Discussed: 'withStrongBorder' is used because there is already a normal border on the arrow component